code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
Thing is, then there's no incentive to play security over detective. They're two distinctive roles; merging their duties in such a clear cut way is just repeating the same shit tg ended up with in terms of the geneticist's access. I've no problem with the detective filling the gaps during skeleton crew, but their gear is definitely sufficient to do both jobs. A cell and a stun baton are not required to fulfill those duties.